app/assets/stylesheets/alchemy/forms.scss

```diff
@@ -146,7 +146,7 @@ form {
     }
   }
 
-  .control_group {
+  .control_group, .control-group {
```

Comment:
As part of this PR a bit out of scope. But I could just use the underscore version in this PR